backend/server/server_frontend_not_embed.go

```diff
@@ -21,16 +20,6 @@ func embedFrontend(e *echo.Echo) {
 	})
 }
 
-// In non embedded mode, the redirect URL is the frontend URL which is different
-// from the external URL. By default, this frontend URL is http://localhost:3000
-func oauthRedirectURL(_ string) string {
-	url := os.Getenv("BB_REDIRECT_URL")
-	if url != "" {
-		return url
-	}
-	return "http://localhost:3000"
-}
-
 func oauthErrorMessage(externalURL string) string {
```

Comment:
I guess we can also remove this in embed and non-embed and just putting in the caller